1588.Sum-of-all-odd-length-subarrays.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/sum-of-all-odd-length-subarrays/
+ */
+
+/**
+ * @param {number[]} arr
+ * @return {number}
+ */
+var sumOddLengthSubarrays = function(arr) {
+    
+    let totalNumber = 0;
+    
+    for (let i = 0; i <= arr.length - 1; i++) {
+        for (let j = arr.length - 1; j >= i; j--) {
+            
+            let target = (arr.slice(i, j + 1));
```

Comment:
Can you use const?
you don't nedd first ().
Can you use descriptive name instead of target?

Reply:
I corrected them